### ISSUE TITLE
Proposal: Add a :glob command

### ISSUE THREAD

**Issue by Vifon** [ranger/ranger#1811]
We could add a new `:glob` command that would accept another command and its arguments but glob-expand them first. I imagine it would be implemented in a similar manner to the `:confirm` command from #1765.

**Comment by aRkedos** [comment]
So `:glob delete` would delete all files in the current directory for example, without the need to select them?

**Comment by toonn** [comment]
No, `:glob delete *` would do so. Keep in mind you need to escape filenames for ranger's console.
